cmd/openscap-plugin/xccdf/datastream.go

```diff
@@ -160,6 +160,31 @@ func populateProfileVariables(dsProfile *xmlquery.Node, parsedProfile *xccdf.Pro
 	return parsedProfile, nil
 }
 
+func populateProfileRules(dsProfile *xmlquery.Node, parsedProfile *xccdf.ProfileElement) (*xccdf.ProfileElement, error) {
+	if parsedProfile.Selections == nil {
+		parsedProfile.Selections = []xccdf.SelectElement{}
+	}
+	profileRules, err := getDsElements(dsProfile, "xccdf-1.2:select")
+	if err != nil {
+		return parsedProfile, fmt.Errorf("error finding 'select' elements in profile: %s", err)
+	}
+	for _, rule := range profileRules {
+		ruleIdRef, err := getDsElementAttrValue(rule, "idref")
+		if err != nil {
+			return parsedProfile, fmt.Errorf("error getting value of 'idref' attribute: %s", err)
+		}
+		ruleSelector, err := getDsElementAttrValue(rule, "selector")
```

Comment:
This could be renamed to `ruleSelected` to be aligned to the attribute `selected`. `selector` is the attribute in variables.